docs/schema/codelists.md

```diff
@@ -6,13 +6,8 @@ Codelists can either be open or closed. **Closed codelists** are intended to be
 
 Publishers must use the codes in the codelists, unless no code is appropriate. If no code is appropriate and the codelist is **open**, then a publisher may use a new code outside those in the codelist. If no code is appropriate and the codelist is **closed**, then a publisher is encouraged to create an issue in the [OCDS GitHub repository](https://github.com/open-contracting/standard/issues) about adding a new code.
 
-```{eval-rst}
-.. admonition:: Extending open codelists
-   :class: note
-
-   .. markdown::
-
-      If you use new codes outside those in an open codelist, please document the codes in an [OCDS extension](../guidance/map/extensions) and in your [publication policy](../../guidance/publish/#finalize-your-publication-policy). Please also create an issue in the [OCDS GitHub repository](https://github.com/open-contracting/standard/issues), so that the codes can be considered for inclusion in the codelist.
+```{admonition} Extending open codelists
```

Comment:
Will this admonition display as before, with class note?

Reply:
Hmm, no, it doesn't. https://standard.open-contracting.org/staging/eval-rst/en/schema/codelists/